### ISSUE TITLE
StackScrollTool and stackScrollSynchronizer do not work 

### ISSUE THREAD

**Issue by crysta-l** [cornerstonejs/cornerstoneTools#999]
Hello,I changed version 2 to the latest version, StackScrollTool and stackScrollSynchronizer do not work. 
No error in the console panel，I‘m not sure if there is a problem with my code.

So I put part of my code involved in the codeSandbox -- https://codesandbox.io/s/stackscroll-synchronizer-r0evv

in above example I can use mouseWheel to scroll the series but i can't use left mouse to scroll.

**Comment by crysta-l** [comment]
[here](https://codesandbox.io/s/vue-template-3b7ll) is the example about verstion 2.3.6

**Comment by dannyrb** [comment]
@crysta-l, good catch. I would be more than happy to accept a PR to update the documentation. When using an incorrect tool name, you should see a `warning` in your console that the tool does not exist or was not "added".

For your `scrollSync` issue, you have this snippet:

```js
_this.scrollSyn = new _this.$cornerstoneTools.Synchronizer(
      "cornerstonestackscroll",
      _this.$cornerstoneTools.stackScrollSynchronizer
    );
```
You were using the wrong event. You can import and use our enum, or copy and paste the correct string value: [STACK_SCROLL](https://github.com/cornerstonejs/cornerstoneTools/blob/master/src/events.js#L199)

You also have a load handler that's causing issues on every new image being loaded, and you're adding tools to enabled elements multiple times, CodeSandbox's built in console tends to obscure output. If you load the sandbox's URL directly in chrome, you'll see more helpful output:

![image](https://user-images.githubusercontent.com/5797588/61055415-6e880600-a3bf-11e9-9aca-a835c212a8e5.png)

Updated and working:

Live: https://g2fg5.codesandbox.io/
Editor: https://codesandbox.io/s/stackscroll-synchronizer-g2fg5

**Comment by dannyrb** [comment]
Actually, I'll leave this open until someone nice decides to update the docs with the correct tool name. Bonus points if anyone wants to contribute a StackScrollSync example

**Comment by crysta-l** [comment]
Thank you for your early reply, I didn't see it in time just because I was called to do something else.
